### PR TITLE
Modernize Label Pill kit

### PR DESCRIPTION
**Pull request by KFad11**
Modernizing and creating a spec file for this kit. 

Currently WIP, just needed to get this up to check on my diff's.

### CODE REVIEW

File: spec/pb_kits/playbook/kits/label_pill_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  it { is_expected.to define_partial }
+  it { is_expected.to define_prop(:label) }
+  it { is_expected.to define_prop(:pill_value) }
+  it { is_expected.to define_enum_prop(:variant)
+                      .with_default("neutral")
+                      .with_values("neutral", "success", "warning", "error", "info") }
+  describe "#classname" do
+    it "returns namespaced class name", :aggregate_failures do
+      expect(subject.new({}).classname).to eq "pb_label_pill_kit_neutral"
+      expect(subject.new(classname: "additional_class").classname).to eq "pb_label_pill_kit_neutral additional_class"
```
Comment (eabousaif) [inline spec/pb_kits/playbook/kits/label_pill_spec.rb]: I'd add one more test asserting variant changes for the pills

Comment (KFad11) [reply spec/pb_kits/playbook/kits/label_pill_spec.rb]: I like it! Got the spec updated. Thanks.

File: app/pb_kits/playbook/pb_label_pill/_label_pill.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname("pb_label_pill_kit")) do %>
-  <%= object.label %>
-  <%= object.pill %>
```
Comment (garettarrowood) [inline app/pb_kits/playbook/pb_label_pill/_label_pill.html.erb]: The classname you've provided here does not appear to match what is was before.

File: app/pb_kits/playbook/pb_label_pill/_label_pill.html.erb [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+    class: object.classname) do %>
+      <%= pb_rails("caption", props: { text: object.label }) %>
```
Comment (garettarrowood) [inline app/pb_kits/playbook/pb_label_pill/_label_pill.html.erb]: This was getting passed `classname: "pb_label_pill_label"`

Comment (KFad11) [reply app/pb_kits/playbook/pb_label_pill/_label_pill.html.erb]: got both class names restored to their former glory. Spec still passes as is.

**Pull request merged**